model/src/main/kotlin/config/AnalyzerConfiguration.kt

```diff
@@ -22,6 +22,7 @@ package org.ossreviewtoolkit.model.config
 
 import com.fasterxml.jackson.annotation.JsonInclude
 
+@JsonInclude(JsonInclude.Include.NON_NULL)
```

Comment:
What do you think about annotating properties instead of classes to not make any assumption about the usage from within the class?

Reply:
I don't get the "to not make any assumption about the usage from within the class" part, sorry. Can you elaborate?

Reply:
Sorry, my bad. I got the semantics wrong when used to annotate classes.